### ISSUE TITLE
Unable to find some local files

### ISSUE THREAD

**Issue by sylvaindeloux** [crowdin/crowdin-cli#349]
**Describe the bug**
I can't load local files to push them to Crowdin.

**To Reproduce**
Configuration file:
```
project_id: "xxx"
api_token: "xxx"
base_path: "xxx"
base_url: "https://api.crowdin.com"
preserve_hierarchy: "true"

files:
    -
        source: "/translations/messages+intl-icu.en.yaml"
        translation: "/translations/build/messages+intl-icu.%two_letters_code%.yaml"
    -
        source: "/translations/admin+intl-icu.en.yaml"
        translation: "/translations/build/admin+intl-icu.%two_letters_code%.yaml"
```

Execute cli command:
```
crowdin push
```

See an error:
```
✔️  Fetching project info
❌ No sources found for '/translations/messages+intl-icu.en.yaml' pattern. Check the source paths in your configuration file
❌ No sources found for '/translations/admin+intl-icu.en.yaml' pattern. Check the source paths in your configuration file
❌ Current execution finished with errors
```

**Expected behavior**
The files are present and valid, so the `push` command should detect them.

**Environment:**
 - OS: macOS 11.2.1
 - Crowdin CLI 3.5.4

**Additional context**
When using `source: "/translations/*.yaml"` syntax, the files are found, but that's not the expected behavior.

**Comment by OlenkaCrowdin** [comment]
Hello!
It would be very helpful if you share with us the file if it is possible
Thank you

**Comment by sylvaindeloux** [comment]
**translations/messages+intl-icu.en.yaml:**
```
test: This is a test
```

Also tried with empty file or a "big" one from another Crowdin project, the result is the same.

The problem is the usage of `+intl-icu` string in the filename (syntax required for Symfony projects using ICU: https://symfony.com/doc/current/translation/message_format.html#using-the-icu-message-format)

**Comment by OlenkaCrowdin** [comment]
Could you please confirm that if the file name is without the usage of +intl-icu in the filename everything works well?

**Comment by sylvaindeloux** [comment]
Yes :)

With this config (and the renamed yaml file):
```
    -
        source: "/translations/admin.en.yaml"
        translation: "/translations/build/admin.%two_letters_code%.yaml"
```

I get this result:
```
✔️  Fetching project info
✔️  File 'translations/admin.en.yaml'
```

**Comment by OlenkaCrowdin** [comment]
Wow, it's great :) 

I would recommend using such a file name, and also, I'll pass your case with the usage of +intl-icu in the filename to our tech team (they are out of the office until Monday) for further investigation 

So let's stay tuned and wishing you a lovely weekend :)

**Comment by sylvaindeloux** [comment]
Thanks Olena :)

**Comment by OlenkaCrowdin** [comment]
You are welcome, Sylvain :)

**Comment by OlenkaCrowdin** [comment]
Hi again :)

I've made some testing on my side and with such path (I also named my file as messages+intl-icu.en.yaml):
           "source" : "/files/*.yaml",
everything works properly and file with such name "messages+intl-icu.en. yaml" uploaded successfully

Hope it is helpful for you🤞

**Comment by sylvaindeloux** [comment]
Yes, but unfortunately I'd like to separate my domains `messages` and `admin`

**Comment by OlenkaCrowdin** [comment]
Do you mean that you want to separate your domains "messages" and "admin" at the translation path? If so, you are welcome to change the translation path to: 
 "translation" : "/translations/build/%two_letters_code%/%original_file_name%",
Or I understood you incorrectly?

**Comment by andrii-bodnar** [comment]
Hi @sylvaindeloux!

This issue is fixed and the fix will be available in the next version of CLI (3.5.5). The new version will be released tomorrow approximately at 10 AM UTC.

**Comment by sylvaindeloux** [comment]
Thanks :)
